app/app.yaml

```
@@ -45,12 +49,14 @@ modules:
         # - after auth so that it can access the params from auth;
         - genutil
         - authz
+        - feegrant
         - transfer
         - interchainaccounts
         - feeibc
         - upgrade
         - consensus
         - circuit
+        - feemarket
```

```suggestion
        # feemarket must be after genutil to allow genesis transactions with 0fees
        - feemarket
```

Could be worth to document :)